packages/medusa-plugin-sendgrid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medusa-plugin-sendgrid",
-  "version": "1.3.11",
+  "version": "1.3.12-next",
```

Comment:
todo: could you revert that please, the changeset will mange it

Reply:
agreed